### ISSUE TITLE
Native Balance Teleport Options

### ISSUE THREAD

**Issue by joepetrowski** [polkadot-js/apps#5254]
When connected to the Relay Chain or a parachain that supports teleport, the UI should have a "Send {DOT, KSM} to {RC, ParaId}" option that will construct the necessary extrinsic/XCM to teleport the asset.

Likewise, if the user is connected to a parachain that supports teleport, the UI should recognize if an account has a balance on the Relay Chain and offer the option to teleport some or all of the balance to the parachain.

**Comment by jacogr** [comment]
1. How do we know it "supports teleport"?
2. Actually have not had a gap to even look at anything xcm-related, so have no idea how to construct these extrinsics at all (including the Junctions, the asset locations, et - I know these from the types, aka copy-and-paste only)

TL;DR Will be  a while before I can have a couple of hours to actually sit down and first understand XCM, the data structures and what actually needs to go into them before looking at this. At this point have no feeling for anything in this space, including where to start construction of the transactions

**Comment by jacogr** [comment]
Notes:

relay -> para: 

- MultiLocation:X1(ParaId)
- X1(AccountId32)
- MultiAsset:ConcreteFungible(Id(Null), value)

para -> relay:

- - MultiLocation:X1(Parent)
- X1(AccountId32)
- MultiAsset:Concrete(Id(X1:Parent), value)
